src/signals/incident-management/containers/Dashboard/components/Filter/Filter.test.tsx

```diff
@@ -258,17 +258,17 @@ describe('FilterComponent', () => {
     userEvent.tab({ shift: true })
 
     expect(
-      screen.queryByRole('listbox', {
+      screen.queryByRole('combobox', {
         name: 'Categorie',
       })
     ).toHaveFocus()
 
     act(() => {
       fireEvent.keyDown(
-        screen.getByRole('listbox', {
+        screen.getByRole('combobox', {
           name: 'Categorie',
         }),
-        { code: 'Enter' }
+        { code: 'Space' }
```

Comment:
Why does "enter" not work anymore?

Reply:
its shouldn't check maar https://codepen.io/piyush-tapaniya/pen/MdNJNy

Reply:
It doesn't either in the sandbox